exercises/node-javascript/add-esdk-start/retrieve.js

```diff
@@ -1,20 +1,18 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-/* ADD-ESDK-START:
- * Add the @aws-crypto/client-node dependency.
- * You will need the decrypt stream as well as the KMS keyring.
- * Look at the config file to see how to pull the Faythe CMK.
- * Then pipe the encrypted stream to the decrypt stream: Profit.
- */
+// ADD-ESDK-START: Add the dependency
 
 const { S3 } = require("aws-sdk");
 const s3 = new S3();
 const config = require("./config");
 const Bucket = config.state.bucketName();
 
+// ADD-ESDK-START: Plumb In Your Config
```

Comment:
// ADD-ESDK-START: Set up a keyring to use Faythe's CMK for decrypting.

Reply:
@lizroth What do you think of these comments?  I like them more, but the titles of the various instructions should match.

Thoughts?

Reply:
Strong agree, I was planning to take a crack at them tomorrow with a slightly fresher perspective, but I think we should do them.

https://github.com/aws-samples/busy-engineers-document-bucket/issues/66

Reply:
KK, I will update both the instructions and these comments